tekton/build_pipeline_test.go

```diff
@@ -47,7 +47,7 @@ var _ = Describe("build pipeline", func() {
 					"pipelines.openshift.io/strategy":          "s2i",
 					"appstudio.openshift.io/component":         "component-sample",
 					"build.appstudio.redhat.com/target_branch": "main",
-					"pipelinesascode.tekton.dev/event-type":    "pull_request",
```

Comment:
I think it would be good to keep this label because in most cases it will be present, wdyt?

Reply:
Yes, you are right, these labels and annotations should all exist, thanks! Updated.